hhvm/deb/skeleton/DEBIAN/postinst

```diff
@@ -39,6 +39,7 @@ echo "*    --install /usr/bin/php php /usr/bin/hhvm 60"
 echo "********************************************************************"
 
 install -d -m 755 -o www-data -g www-data /var/run/hhvm
+install -d -m 01733 /var/lib/hhvm/sessions
 
```

Comment:
> install -d -m 01733 /var/lib/hhvm/sessions

btw, this will make parent dirs created with default umask!

so if /var/lib/hhvm dir does not exist, and you create /var/lib/hhvm/sessions with umask 027 then /var/lib/hhvm dir permission would be 750, not 755:

you must install -d all parent dirs as well to avoid such surprise, or reset to sane umask (i don't know maybe debian installer resets umask for postinst script automaticaly or not)